type_inference_visitor.py

```
@@ -0,0 +1,66 @@
+import warnings
```

Don't use this library; you can raise an exception for now.